samples/shields/lmp90100_evb/rtd/src/main.c

```diff
@@ -94,6 +94,6 @@ void main(void)
 						resistance));
 		}
 
-		k_sleep(1000);
+		k_sleep(K_MSEC(1000));
```

Comment:
Similar thing here as what @galak pointed out elsewhere. I.e. some manual work is needed for things that the script doesn't do the most optimal translation for.

Reply:
> Similar thing here as what @galak pointed out elsewhere. I.e. some manual work is needed for things that the script doesn't do the most optimal translation for.

That's true, but I only signed on to update the script.  I'm not going to add manual changes to make it look prettier than what the script produces.

Reply:
Is it ok if I push an update to this PR for that? It's just four lines that need changing, if I counted right.

Reply:
I prefer commits that contain automatically transformed code to be standalone so they can be verified and updated if the script changes, so if you want to clean it up I'd like to see it as a follow-on commit rather than modifications to the existing ones.  Personal preference, there are no rules/policies that I know of.